doc/installation/pip.rst

```diff
@@ -6,18 +6,15 @@ Installing with pip
 You can install glue along with **all** required and optional dependencies
 with `pip <https://pip.pypa.io/en/stable/>`__ using::
 
-    pip install glueviz[all,qt]
+    pip install glueviz[qt]
 
 The above will include domain-specific plugins as well as the PyQt5 library.
-If you only want to install glue with all required and only non-domain-specific
-optional dependencies (for example excluding the optional dependencies for
-astronomy), you can do::
+If you only want to install glue with minimal dependencies, you can instead
+install individual glue packages, e.g.::
 
-    pip install glueviz[recommended,qt]
+    pip install glue-core   # base package with no front-end
+    pip install glue-qt[qt] # Qt-based front-end
+    pip install glue-vispy-viewers  # 3D viewers
 
-And finally, if you don't want to install optional dependencies at all, and
-already have PyQt5 or PySide2 installed, you can do::
-
-    pip install glueviz
-
-Note that this will still installed required dependencies.
+The ``[qt]`` ensures PyQt5 gets installed, but you can also install PyQt6 or
+PySide yourself and miss out the ``[qt]``.
```

Comment:
```suggestion
    pip install PyQt6 glue-qt # Qt6-based front-end
The ``[qt]`` ensures PyQt5 gets installed and set as the front-end,
but you can also use **instead** an already installed PyQt6 or PySide
front-end, or install it along the glue libraries as above (omitting the ``[qt]``).
```
Perhaps clarifying the different options a bit better here? Or maybe we should link/refer to the qtpy documentation how the default front-end is picked or prescribed setting `QT_API`?

Reply:
I agree we can expand this a bit more - for now I've pushed an update along the lines you suggest, then let's get this PR in so the instructions are correct, then we can do a follow-up with some improvements.